azure-pipelines.yml

```diff
@@ -89,6 +89,8 @@ extends:
     pool:
       name: VSEngSS-MicroBuild2022-1ES
       os: windows
+    settings:
+      skipBuildTagsForGitHubPullRequests: true
```

Comment:
Just wanted to comment on why this line is needed. See https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/sourceanalysisstage and scroll all the way down.

The 1ESPT can't tag builds that are for forks of github repos. So this line disables the tagging.